cJSON.c

```diff
@@ -558,22 +558,22 @@ static cJSON_bool print_number(const cJSON * const item, printbuffer * const out
     /* This checks for NaN and Infinity */
     if (isnan(d) || isinf(d))
     {
-        length = sprintf((char*)number_buffer, "null");
+        length = snprintf((char*)number_buffer, sizeof(number_buffer), "null");
```

Comment:
What is the justification for having this be a `noup` patch? Did upstream reject it? If so, what is wrong with their justification; if not, can we send this upstream and keep it here as a `fromlist` patch instead?

Reply:
Never mind, I see the note in the PR description that explains why upstream probably won't want this.

Can you please put it into the commit log, so we don't lose this information?

Reply:
Updated the commit log explaining why this patch cannot go upstream.